packages/shared-ux/button_toolbar/src/popover/popover.stories.tsx

```diff
@@ -6,36 +6,38 @@
  * Side Public License, v 1.
  */
 
-import { EuiContextMenu } from '@elastic/eui';
-import { ButtonContentIconSide } from '@elastic/eui/src/components/button/button_content';
-import { Story } from '@storybook/react';
 import React from 'react';
-import { ToolbarPopover } from './popover';
-import mdx from './popover.mdx';
+
+import { EuiContextMenu } from '@elastic/eui';
+
+import { ToolbarPopover as Component } from './popover';
+import mdx from '../../README.mdx';
+
+const argTypes = {
+  iconSide: {
+    control: {
+      type: 'radio',
+      options: ['left', 'right', 'undefined'],
+    },
+  },
+};
+
+type Params = Record<keyof typeof argTypes, any>;
 
 export default {
-  title: 'Toolbar/Popover',
+  title: 'Solution Toolbar',
   description: 'A popover that is a part of a toolbar.',
   parameters: {
     docs: {
       page: mdx,
     },
   },
-  argTypes: {
-    iconSide: {
-      control: {
-        type: 'radio',
-        options: ['left', 'right', 'undefined'],
-      },
-    },
-  },
+  argTypes,
 };
 
-export const Component: Story<{
-  iconSide: ButtonContentIconSide | undefined;
-}> = ({ iconSide }) => {
+export const Popover = ({ iconSide }: Params) => {
   return (
-    <ToolbarPopover label="Add element" iconType="plusInCircle" iconSide={iconSide}>
+    <Component label="Add element" iconType="plusInCircle" iconSide={iconSide}>
```

Comment:
Better convention for icons vs sides and fixes the padding issue:

```suggestion
    <Component label="Add element" iconType={iconSide === 'right' ? 'arrowDown' : 'plusInCircle'}  iconSide={iconSide} panelPaddingSize="none">
```